spec/lib/figure/pawn_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'figure/pawn'
+
+describe Figure::Pawn do
```

Comment:
you can look at alternative way of solving that challenge:

https://github.com/cs3b/chess/pull/3#discussion_r70788578